test/interactiveutil.jl

```diff
@@ -0,0 +1,110 @@
+### Testing download(url,[localfile],[silent=false])
+
+"Runs an expression, and returns stdout, stderr, and the result."
+function capture_output(expression)
+	originalSTDOUT = STDOUT
+	originalSTDERR = STDERR
+	(errRead, errWrite) = redirect_stderr()
+	(outRead, outWrite) = redirect_stdout()
+
+	result = eval(expression)	
+	# Ensure both streams are written to or readavailable waits forever.
+	print("\n")
+	write(STDERR, "\n")
+
+	close(errWrite)
+	close(outWrite)
+	errors = readavailable(errRead)
+	output = readavailable(outRead)
+	close(outRead)
+	close(errRead)
+	redirect_stderr(originalSTDERR)
+	redirect_stdout(originalSTDOUT)
+
+	errstr = join(map(x->Char(x), errors))
+	outstr = join(map(x->Char(x), output))	
+	# Remove the manually added \n from stdout and stderr
+	return (chop(outstr), chop(errstr), result)
```

Comment:
Should be using [`chomp`](http://docs.julialang.org/en/latest/stdlib/strings/#Base.chomp) rather than [`chop`](http://docs.julialang.org/en/latest/stdlib/strings/#Base.chomp) here